lib/galaxy/webapps/galaxy/buildapp.py

```diff
@@ -128,6 +129,9 @@ def paste_app_factory( global_conf, **kwargs ):
 
     register_postfork_function(postfork_setup)
 
+    for th in threading.enumerate():
+        if th.is_alive():
+            log.debug("Prior to webapp return, Galaxy thread %s is alive.", th)
```

Comment:
@natefoo Ok, cool, won't wrap it.  I did find a bug though.  uwsgidecorators was *not* in my environment but uwsgi was.  The try/except was masking this error and stating that process_is_uwsgi was false, even though it was true.  The fix in efbd8f2 resolves that, but I'm trying to figure out why the decorator package is (now?) separate and not found.